Src/Base/AMReX_Math.H

```diff
@@ -109,7 +109,7 @@ std::pair<double,double> sincos (double x)
 {
     std::pair<double,double> r;
 #if defined(AMREX_USE_SYCL)
-    r.first = sycl::sincos(x, &r.second);
+    r.first = sycl::sincos(x, sycl::multi_ptr<double,sycl::access::address_space::global_space>(&r.second));
```

Comment:
Thanks for spotting this!

Looks like the ctor of multi_ptr is explicit.
```
explicit multi_ptr(typename multi_ptr<ElementType, Space, access::decorated::yes>::pointer);
```
So the conversion is not supposed to happen.  But is `global_space` the right address space?  `r` is a thread private variable likely in the register  So what does it mean to create a multi_ptr in global space from its address?

Reply:
Fair point, I've done some testing, saw no performance difference but I think semantically using the local address space makes more sense like you suggested :)

Reply:
Private space is more appropriate.